packages/twenty-front/src/modules/settings/security/utils/guessIconByUrl.tsx

```diff
@@ -0,0 +1,9 @@
+import { IconComponent, IconGoogle, IconKey } from 'twenty-ui';
+
+export const guessIconByUrl = (url: string): IconComponent => {
```

Comment:
same here, we want unique names to ease dev xp (auto impots) and this will likely conflict with another file at some point